distortion_model: "plumb_bob"
D: []
K: [2215.13350577, 0.0, 1030.47471121, 0.0, 2215.13350577, 756.735726174, 0.0, 0.0, 1.0]
